src/ipfsd-daemon.js

```diff
@@ -80,77 +72,56 @@ class Daemon {
     this.api.gatewayPort = this.gatewayAddr.nodeAddress().port
   }
 
-  /**
-   * Current repo path
-   *
-   * @member {string}
-   */
-  get repoPath () {
-    return this.path
-  }
-
-  /**
-   * Shell environment variables
-   *
-   * @member {object}
-   */
-  get env () {
-    return this.path ? Object.assign({}, this._env, { IPFS_PATH: this.path }) : this._env
-  }
-
   /**
    * Initialize a repo.
    *
-   * @param {Object} [initOptions={}]
-   * @param {number} [initOptions.bits=2048] - The bit size of the identiy key.
-   * @param {string} [initOptions.directory=IPFS_PATH] - The location of the repo.
-   * @param {string} [initOptions.pass] - The passphrase of the keychain.
+   * @param {Object} [initOptions={}] - @see https://github.com/ipfs/js-ipfs/blob/master/README.md#optionsinit
    * @returns {Promise}
    */
   async init (initOptions) {
-    if (this.initialized && initOptions) {
+    if (this.initialized && typeof initOptions === 'object') {
```

Comment:
`initOptions` can now be a boolean or an object?  Why is that?

Reply:
yes this version only supports options exactly in the same way js-ipfs does, ill rewrite the description of this PR explaining everything when i finish the tests